src/main/java/io/github/terra121/util/AzimuthUtils.java

```diff
@@ -0,0 +1,35 @@
+package io.github.terra121.util;
+
+public class AzimuthUtils {
```

Comment:
this should be a lombok `@UtilityClass`.

that said, there's no reason to add an entire class for one static utility method... move `azimuthToFacing` to `TerraUtils`.

Reply:
Or maybe to GeographicProjection.